supporting_content/dimensions/usageunit.md

```diff
@@ -0,0 +1,20 @@
+# Usage Unit
+
+## Example provider mappings 
```

Comment:
Consider adding a note or disclaimer that these columns are a close match. Direct mapping is not recommended, especially for AWS and Microsoft, not only due to the Unit Format requirement/suggestion but also because of pricing rules such as block pricing.

Reply:
> but also because of pricing rules such as block pricing.

Sorry, trying to understand what you mean by this.

Reply:
Added comment